python/run_benchmark.sh

```diff
@@ -66,7 +67,7 @@ knn_num_rows=$num_rows
 num_cols=${num_cols:-3000}
 
 # for large num_rows (e.g. > 100k), set below to ./benchmark/gen_data_distributed.py and /tmp/distributed
-gen_data_script=./benchmark/gen_data.py
+gen_data_script=${gen_data_script:-./benchmark/gen_data.py}
```

Comment:
Is there a different gen_data script that should be included in this PR?  Or is this just a hook for debugging?